services/core/shared/indexdb/mysql.js

```diff
@@ -165,7 +165,9 @@ const getDbInstance = async (tableName, tableConfig, connEndpoint = config.endpo
 
 		if (params.orWhere) {
 			const { orWhere } = params;
-			query.where(queryParams).orWhere(orWhere);
+			query.where(function () {
```

Comment:
Does it fail on linter? Wouldn't `() => {}` work as well?

Reply:
It is not possible to switch it to `arrow functions` in this case because Arrow functions doesn't have their own `this` binding and we need to use `this` for the results. Also, it doesn't fail on linter so we are good now.